lib/conditionals.md

```diff
@@ -5,8 +5,8 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional uses the statements 'if', 'elsif' and 'else' to move the otherwise linear progression of a program in an alternative direction based on the true-ness or false-ness of the conditional statement. 
```

Comment:
It is correct that `if` (and its cousins) are used with conditionals to "move the otherwise linear progression of a program..." (a term that describes that is "control flow", by the way).

There is _another_ common usage of conditionals though, related to control flow. What is it?